docker-compose.ci.yml

```diff
@@ -16,6 +16,13 @@ services:
       - db
       - cache
     command: /obs/contrib/start_minitest
+  lint:
```

Comment:
Why?

Reply:
Why what?

Reply:
This is unrelated to CircleCI. The docker-compose.ci.yml file is not used in CircleCI. Also we already have a Rake task for this: rake docker:test:lint which uses the frontend container. Also you don't use these changes anywhere ....

Reply:
ups this slipped in from a test. good catch, sorry